spec/integ/admin-rooms.spec.js

```diff
@@ -308,6 +309,77 @@ describe("Admin rooms", function() {
         expect(sentSay).toBe(true, "sent say IRC command");
     }));
 
+    it("should be able to keep their name using !nick",
+        test.coroutine(function* () {
```

Comment:
We prefer async functions

```suggestion
        async () {
```

Reply:
`test.coroutine(function* ` stays consistent with all other tests in the file. May I find-replace it to async everywhere then?

Reply:
sounds like a separate short PR to me...